contrib/images/simd-env/Dockerfile

```diff
@@ -12,7 +12,7 @@ VOLUME [ /simd ]
 WORKDIR /simd
 EXPOSE 26656 26657
 ENTRYPOINT ["/usr/bin/wrapper.sh"]
-CMD ["start"]
+CMD ["start", "--log_format", "plain"]
```

Comment:
For local networks that are used for testing, human readable/friendly logs are preferable.